channels/rdpdr/server/rdpdr_main.c

```diff
@@ -725,13 +805,12 @@ static BOOL rdpdr_server_send_user_logged_on(RdpdrServerContext* context)
 	status = WTSVirtualChannelWrite(context->priv->ChannelHandle, (PCHAR) Stream_Buffer(s), Stream_Length(s), &written);
 
 	Stream_Free(s, TRUE);
-
-	return status;
+	return status ? ERROR_SUCCESS : ERROR_INTERNAL_ERROR;
```

Comment:
I was wondering if there is a reason why ERROR_SUCCESS is used here instead of CHANNEL_RC_OK.